website/source/docs/http/sys-root-generation.html.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: "http"
+page_title: "HTTP API: /sys/root-generation/"
+sidebar_current: "docs-http-sys-root-generation"
+description: |-
+  The `/sys/root-generation/` endpoints are used to create a new root key for Vault.
+---
+
+# /sys/root-generation/attempt
```

Comment:
This is a super nit, but we usually start with the verb (e.g. revoke, revoke-prefix, etc). Can we do `/sys/generate-root/` as the prefix?